services/core-api/app/api/mines/permits/permit/models/permit.py

```diff
@@ -209,3 +226,32 @@ def validate_permit_no(self, key, permit_no):
         if not permit_no:
             raise AssertionError('Permit number is not provided.')
         return permit_no
+
+    @classmethod
+    def validate_exemption_fee_status(cls,
+                                      is_exploration,
+                                      permit_status,
+                                      permit_prefix,
+                                      mine_disturbance_codes=None,
+                                      mine_tenure_type_codes=None,
+                                      exemption_fee_status_code=None):
+        current_app.logger.debug(is_exploration)
```

Comment:
Might be better to remove this debug or turn it into a more useful statement

Reply:
oops. removed.

Reply:
done